### DISCUSSION TITLE
Useful snippets and tips

### DISCUSSION THREAD

**Discussion by BigJk** [BigJk/snd#63] [Show and tell]
You can share useful snippets and tips for S&D here 😄

**Comment by MrG4dget** [comment]
Just in case someone can use this:

Wanted to create a QR code that opens the URL to the dndbeyond entry of a spell. Got it working with the following:

```html 
<img style="display: block; margin-left: auto; padding: 20px;  margin-right: auto; width: 50%;" 
src="{{"https://api.qrserver.com/v1/create-qr-code/?size=150x150&data="+"https://www.dndbeyond.com/spells/"+config.name | replace(" ","-") | lower+""}}"></img>
```

So if the config has the Spells name, it should create the correct QR code.
![Screenshot 2024-05-20 160019](https://github.com/BigJk/snd/assets/3408565/1ebfcf96-25b1-4f42-9672-a58cbc965b1f)
